cmd/zoekt-sourcegraph-indexserver/sg.go

```diff
@@ -129,6 +140,94 @@ type sourcegraphClient struct {
 	configFingerprintReset time.Time
 }
 
+// GetRepoRank asks Sourcegraph for the score vector of repoName.
+func (s *sourcegraphClient) GetRepoRank(ctx context.Context, repoName string) ([]float64, error) {
```

Comment:
this is unused right? I suppose will be used soon.

Reply:
With the introduction of []float64, the repo rank is not as straightforward to incorporate as a simple float64. I want to do that in a separate PR.